core/lib/types/src/contract_verification/contract_identifier.rs

```diff
@@ -58,28 +64,150 @@ impl DetectedMetadata {
     pub fn length(self) -> usize {
```

Comment:
Nit: since the type is no longer `Copy`, this method should take `&self` instead of `self`.

Reply:
Fixed